src/eterna/AnnotationManager.ts

```diff
@@ -927,24 +901,30 @@ export default class AnnotationManager {
      * @param item display object data with positioning and annotation metadata
```

Comment:
Subtle change, but it's probably worth adding `reset` and `ignoreCustom` to the method signature

Reply:
Oh yep good call!